src/app/client/src/app/modules/observation/components/alert-modal/alert-modal.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="sb-modal">
```

Comment:
we have a modal component already in the application please use that instead of creating a new alert modal

Reply:
We needed a modal that needs to be triggered from multiple pages, without copy-pasting the HTML. 

Is there any modal that can be triggered from the service?